build/Uno.UI.WpfHost.nuspec

```diff
@@ -16,7 +16,7 @@
 	<dependencies>
 
 	  <group targetFramework="net462">
-		<dependency id="Uno.UI" version="1.42.0-dev.395" />
+		<dependency id="Uno.UI" version="1.43.0-dev.965" />
```

Comment:
Should be 1.43.1-dev.XXX instead of 1.43.0-dev.XXX

Reply:
This file is automatically updated by the build:

https://github.com/nventive/Uno/blob/4ad2327e8f0189bbe75f7b0e08365f7e73331c65/build/Uno.UI.proj#L135